AsyncDisplayKit/Private/ASDisplayNode+FrameworkPrivate.h

```diff
@@ -49,7 +49,9 @@ typedef NS_OPTIONS(NSUInteger, ASHierarchyState)
   /** One of the supernodes of this node is performing a transition.
       Any layout calculated during this state should not be applied immediately, but pending until later. */
   ASHierarchyStateLayoutPending           = 1 << 3,
-  ASHierarchyStateVisualizeLayout         = 1 << 4
+  ASHierarchyStateYogaLayoutEnabled       = 1 << 4,
```

Comment:
Likewise, I could #define these values and "Illegally" OR them into the state value.  However since it doesn't cause tangling, it seemed best to add these directly.